plugins/star-rating/frontend/public/javascripts/countly.views.js

```diff
@@ -1307,7 +1291,16 @@ window.starView = countlyView.extend({
                 })
                 $(this).addClass('star-rating-tab-item-active');
                 self._tab = $(this).data('target');
-                window.location.hash = '/' + countlyCommon.ACTIVE_APP_ID + '/analytics/star-rating/' + $(this).data('target');
+                //window.location.hash = '/' + countlyCommon.ACTIVE_APP_ID + '/analytics/star-rating/' + $(this).data('target');
+                if(history.replaceState) {
+                    history.replaceState(null, null, '#/' + countlyCommon.ACTIVE_APP_ID + '/analytics/star-rating/' + $(this).data('target'));
+                }
+                else {
+                    location.hash = '#/' + countlyCommon.ACTIVE_APP_ID + '/analytics/star-rating/' + $(this).data('target');
+                }
```

Comment:
I think for this, we have this:
https://github.com/Countly/countly-server/blob/master/frontend/express/public/javascripts/countly/countly.template.js#L2671